src/main/resources/com/google/api/codegen/nodejs/standalone_sample.snip

```diff
@@ -238,6 +245,18 @@
     });
 @end
 
+@private methodCallSampleCodeLongRunningPromiseAwait(apiMethod, sample)
+  {@initCode(apiMethod, sample.sampleInitCode)}
+
+  // Create a job that you can wait for now, or get its result later
```

Comment:
"Create a job whose results you can either wait for now, or get later"

Reply:
BTW, I assume initial capitalization and no final period is the comment style we're going for?

Reply:
idk, I literally copied the comments over from the example Rebecca gave me :/

Reply:
I would still tweak the wording as per above. The original was not quite structurally clear English....

Reply:
Oh yeah for sure.